docs/variants/asus_kgpe_d16/setup.md

```diff
@@ -0,0 +1,214 @@
+RTE v.0.5.3 with Asus KGPE-D16 connection
+=========================================
+
+![](/images/kgpe_overview.png)
+
+#### Power supply
+
+Power to the Asus KGPE-D16 is delivered by common PC power supply. Simply
+connect the power supply to ATX compatible connectors on the mainboard
+(SSIPWR1, SSI12V1, SSI12V2).
+
+#### SPI
+
+BIOS SPI flash is a socketed DIP-8. There is no simple way to attach to it with
+any tools. That is why we have thought about the
+[qspimux](https://github.com/felixheld/qspimux) made by Felix Held which is
```

Comment:
@miczyg1 I'm not sure if this section should not be in transparent validation somewhere and just linked here. So extensive description is important, but it does not belong to KGPE-D16 IMO.

Reply:
Nowhere else do we use the qspimux besides the KGPE-D16, I don't see any reason to move it elsewhere. qspimux helps to update the flash content and does not have any validation purpose. Moreover qspimux is customizable and can be used with a variety of chips and adapters, so putting it into some generic section will only cause a scatter of information and jumping throughout the repository.

Reply:
@miczyg1 what about potential reuse of information?

Reply:
@miczyg1 let's agree if any other project will use this tool, then you will move the documentation to generic section, so both hardware platforms will not cross refere itself, but will point to generic content, ok?

Reply:
Potential reuse of information existing in this document is just a few sentences (2-3). If it grows and appears in other systems then I'm fine with that approach.